src/Components/Scenes/RequestAssets/RemoveModal/RemoveModal-Container.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { ListItem } from '@material-ui/core';
+import weight from '@edma/design-tokens/js/weight';
+import RemoveModal from '../../../Presentational/RequestAssets/RemoveModal';
+import {
+  handleModalOpen,
+  handleModalClose
+} from '../../../../Actions/requestAssetActions';
+
+const mapStateToProps = ({ requestAssets }) => {
+  return {
+    modalText:
+      'Are you sure that you want to remove the following assets from your request?',
+    selectedSearchResultCopy: requestAssets.selectedSearchResultCopy,
+    render: requestAssets.selectedSearchResultCopy
```

Comment:
What would be your thoughts on moving this functionality within the component itself? Maybe then it could utilize the `selectedSearchResultCopy` prop? Perhaps creating a function like `createRequestsList` or something to do this.  Then it would be possible to move `renderStyle` to the component as well then.

I guess this all depends on whether there are plans to re-use this component at all.

Reply:
The reason I need to move my JSX into container  is to make reusable Component. Cuz `Table Modal and Notification` will be using in different place though-out the application. 
I spend some time make move the JSX somewhere but couldnt fine better solution. Let me know if you have an idea.

Reply:
Ahh yeah sure, if it's meant to be a reusable component across the entire app then I would suggest a more generic naming convention and we could move that to a different location, perhaps a Common directory within Presentational?

@robertwollny-pnmac @abbyjb @Jake-Lindvall would love to hear your thoughts on how we could handle truly generic components to be reused app-wide.

Reply:
@lheld-pnmac  I moved file into `Presentational` folder under the feature name Folder. The reason I didnt put into Common directory. In the future Common direcotry might have lots of file in it. Become too flat too look for something

For naming convention. I open the suggestions.

Reply:
Sounds good! Let's stick with this for now and as the application grows, we can break out those common components into their own directory for reuse if it comes up. 👍